extensions/arc/images/Icon-general-12-Billing.svg

```diff
@@ -0,0 +1,17 @@
+<svg id="b82fda19-b66e-482f-99d7-d23ab8197f70"  xmlns="http://www.w3.org/2000/svg"  width="18" height="18" viewBox="0 0 18 18">
```

Comment:
Let's try to give these shorter and more descriptive names

Reply:
I'll shorten them.  These long filenames are how they were downloaded from https://cds-icons.azurewebsites.net